backend/core/src/auth/services/user.service.ts

```diff
@@ -116,16 +114,9 @@ export class UserService {
     const qb = this._getQb()
 
     if (params.filter) {
-      addFilters<Array<UserFilterParams>, typeof userFilterTypeToFieldMap>(
-        params.filter,
-        userFilterTypeToFieldMap,
-        distinctIDQB
-      )
-      addFilters<Array<UserFilterParams>, typeof userFilterTypeToFieldMap>(
-        params.filter,
-        userFilterTypeToFieldMap,
-        qb
-      )
+      const filter = new UserQueryFilter()
+      filter.addFilters(params.filter, userFilterTypeToFieldMap, distinctIDQB)
+      filter.addFilters(params.filter, userFilterTypeToFieldMap, qb)
```

Comment:
In this case, we want to keep what is in Detroit (we have a new issue to update core with this).